docs/design/autoware-interfaces/ad-api/list/api/fail_safe/index.md

```diff
@@ -1,20 +1,41 @@
 # Fail-safe API
 
-- {{ link_ad_api('/api/fail_safe/state') }}
+- {{ link_ad_api('/api/fail_safe/mrm/state') }}
```

Comment:
@kenji-miyake @mkuri 
Renamed https://github.com/autowarefoundation/autoware-documentation/pull/165/commits/291883e6c33e2df91fa759cc893a017f906141ba. Or should I consider APIs such as:

```
/api/fail_safe/mrm/activate_confortable_stop
/api/fail_safe/mrm/activate_suddon_stop
```

Reply:
Thank you! I feel it's natural if we drop `mrm/` for the APIs.

```
/api/fail_safe/activate_confortable_stop
/api/fail_safe/activate_suddon_stop
```

Reply:
I think it's good since it seems unlikely that anything other than MRM will direct the behavior as fail_safe.
`/api/fail_safe/activate_confortable_stop
/api/fail_safe/activate_suddon_stop`

Reply:
Done https://github.com/autowarefoundation/autoware-documentation/pull/165/commits/291883e6c33e2df91fa759cc893a017f906141ba.